docs: `r/vsphere_virtual_machine` updated disks info (#2272)
hashicorp/terraform-provider-vsphere: website/docs/r/virtual_machine.html.markdown

The following requirements apply to import:

* The disks must have a [`label`](#label) argument assigned in a convention matching `Hard Disk`, starting with disk number 0, based on each virtual disk order on the SCSI bus. As an example, a disk on SCSI controller `0` with a unit number of `0` would be labeled as `Hard Disk 0`, a disk on the same controller with a unit number of `1` would be `Hard Disk 1`, but the next disk, which is on SCSI controller `1` with a unit number of `0`, still becomes `Hard Disk 2`.

> ~> **NOTE:** Any custom `label` set at deployment of machine through terraform, on import will  not have custom `label`, it will default back to `Hard Disk x`.

Reviewer: ```suggestion
~> **NOTE:** Any custom `label` set at deployment of machine through Terraform, on import will not have the custom `label` and will default to `Hard Disk _x_`.
```
